AirCasting/Extensions/SensorThreshold.swift

```diff
@@ -0,0 +1,48 @@
+// Created by Lunar on 30/04/2021.
+//
+
+import Foundation
+import SwiftUI
+
+extension SensorThreshold {
+    
+    var rawThresholdsBinding: Binding<[Float]> {
+        Binding<[Float]> { [self] in
+            [
+                Float(thresholdVeryLow),
+                Float(thresholdLow),
+                Float(thresholdMedium),
+                Float(thresholdHigh),
+                Float(thresholdVeryHigh)
+            ]
+        } set: { [self] newThresholds in
+            guard newThresholds.count >= 5 else { return }
+            thresholdVeryLow = Int32(newThresholds[0])
+            thresholdLow = Int32(newThresholds[1])
+            thresholdMedium = Int32(newThresholds[2])
+            thresholdHigh = Int32(newThresholds[3])
+            thresholdVeryHigh = Int32(newThresholds[4])
+        }
+    }
+}
+
+extension SensorThreshold {
+    // mocked SensorThreshold is used for previews (& temporarily some views)
+    static var mock: SensorThreshold {
```

Comment:
This should be avoided. If needed for preview should be wrapped in `#if DEBUG` and if needed for development should not end up in merged develop code. just by referencing without context save possible to create garbage value that could cause random failures in random places that could lead to dead end investigation. Also if needed that means a protocol should lld be created and mocks unreleased to core data could be made. All in all please avoid at all costs. Even if needed for at start for feature branch development please start with `@available(*, deprecated, renamed: "DEVELOPMENT ONLY")`  annotation

Reply:
Thanks for the tip, I'll be doing that in the future. 
I added `#if DEBUG ` here.